csr/gen_csr_value.py

```diff
@@ -16,7 +16,7 @@ def gen_raw_field_type(field_len: int) -> str:
     return "uint64_t"
 
 class CSRField:
-    def __init__(self, name: str, field_descr: dict | list):
+    def __init__(self, name: str, field_descr):
```

Comment:
Such hints are avaliable since python 3.10
Maybe add python version checks and keep this?

Reply:
according to @thelimar knowledge, it is not usefull in runtime. lets leave a TODO comment for now